tests/test_fsspec_compat.py

```diff
@@ -16,13 +16,13 @@ def airtemp_ds():
 
 
 def test_get_zmetadata_key(airtemp_ds):
-    mapper = TestMapper(airtemp_ds.rest.app)
+    mapper = TestMapper(Rest(airtemp_ds).app)
```

Comment:
Is there a reason we can't use the accessor anymore or are you just being more explicit?

Reply:
Yeah it just for being more explicit and closer to the logic tested.